README.md

```diff
@@ -293,10 +293,20 @@ wish to make requests to it
 
 Unlock the vehicle.
 
+#### Return
+| Type               | Description         |
+|:------------------ |:--------------------|
+| Dictionary.`status`| Set to success on successful request. |
+
 ### `lock(self)`
 
 Lock the vehicle.
 
+#### Return
```

Comment:
can you add a `throws` argument so the reader knows what happens on an unsuccesful request